hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2254,4 +2256,70 @@ boolean snapshotCleanupSwitch(final boolean on, final boolean synchronous)
    */
   boolean isSnapshotCleanupEnabled() throws IOException;
 
+  /**
+   * Gets group info for the given group name
+   * @param groupName the group name
+   * @return group info
+   * @throws IOException if a remote or network exception occurs
+   */
+  RSGroupInfo getRSGroupInfo(String groupName) throws IOException;
```

Comment:
So, this an hbase3 targetted patch?

Reply:
It is for branch HBASE-22514, I think all pathes on it can be applied to branch2+.